src/main/resources/dsl/workflow1.yaml

```diff
@@ -0,0 +1,30 @@
+# This sample workflow execute 3 steps in sequence.
+# 1) sampleActivity1, takes arg1 as input, and put result as result1.
+# 2) sampleActivity2, takes result1 as input, and put result as result2.
+# 3) sampleActivity3, takes args2 and result2 as input, and put result as result3.
+
+variables:
+  arg1: value1
+  arg2: value2
+
+root:
+  sequence:
+    elements:
+      - activity:
+          name: SampleActivity1
+          method: SampleActivity1.getInfo
```

Comment:
I had to add the activity name to the method, since there is an issue with the validation registration method where it does not account for the activity name prefix when making sure no two activity/method have the same name.  
![Uploading image.png…]()

Reply:
The reason is that you specify the same name for all the activities. When the name is specified explicitly the prefix is ignored.

Reply:
thanks, did not know that.